metis-indexing/metis-indexing-client/src/main/resources/indexing.properties.example

```diff
@@ -0,0 +1,12 @@
+mongo.host=host1,...,hostn

```

Comment:
Please separate the blocks with comments, by grouping the configuration e.g. Mongo, Solr etc.

Reply:
This is not solved yet.

Reply:
Fixed.